benchmarks/cupy/bench_core.py

```diff
@@ -0,0 +1,20 @@
+from benchmarks.numpy.common import Benchmark
+from benchmarks.utils import sync
+from benchmarks.utils import parameterize
+
+import cupy
+import numpy
+
+
+@sync
+@parameterize([
```

Comment:
How about parameterizing dtypes?

Reply:
Ok. I only added `['float32', 'complex128']` as there should be no much difference of time between dtypes.